{"timestamp": "2025-06-08T00:03:56.456651", "level": "INFO", "logger": "test_main_py", "message": "main.py結合テスト開始", "module": "logger", "function": "_log", "line": 201}
{"timestamp": "2025-06-08T00:03:56.463829", "level": "ERROR", "logger": "test_main_py", "message": "main.py結合テスト: NG invalid literal for int() with base 10: ''", "module": "logger", "function": "_log", "line": 201, "exception": "Traceback (most recent call last):\n  File \"N:\\python_projects\\check_company_with_web\\tests\\test_main.py\", line 20, in test_main_py\n    main()\n  File \"N:\\python_projects\\check_company_with_web\\main.py\", line 24, in main\n    config = load_config()\n             ^^^^^^^^^^^^^\n  File \"N:\\python_projects\\check_company_with_web\\config.py\", line 16, in load_config\n    \"MAX_PROCESSING_TIME\": int(os.getenv(\"MAX_PROCESSING_TIME\", 600)),  # 追加\n                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nValueError: invalid literal for int() with base 10: ''"}
{"timestamp": "2025-06-08T00:04:14.341053", "level": "INFO", "logger": "test_main_py", "message": "main.py結合テスト開始", "module": "logger", "function": "_log", "line": 201}
{"timestamp": "2025-06-08T00:10:59.701342", "level": "INFO", "logger": "test_main_py", "message": "main.py結合テスト: OK", "module": "logger", "function": "_log", "line": 201}
{"timestamp": "2025-06-08T00:12:49.374686", "level": "INFO", "logger": "test_main_py", "message": "main.py結合テスト開始", "module": "logger", "function": "_log", "line": 201}
{"timestamp": "2025-06-08T00:12:57.420187", "level": "INFO", "logger": "test_main_py", "message": "main.py結合テスト: OK", "module": "logger", "function": "_log", "line": 201}
{"timestamp": "2025-06-08T00:13:42.323755", "level": "INFO", "logger": "test_main_py", "message": "main.py結合テスト開始", "module": "logger", "function": "_log", "line": 201}
{"timestamp": "2025-06-08T22:53:58.218457", "level": "INFO", "logger": "test_main_py", "message": "main.py結合テスト開始", "module": "logger", "function": "_log", "line": 201}
